.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,16 @@
 <!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
 <!-- The text between the arrows are comments - they will not be visible on your PR. -->
 
-## About the PR
-<!-- What does it change? What other things could this impact? -->
+## What
+<!-- What did you change in this PR? -->
 
+## Why
+<!-- What was changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
```

Comment:
`What was changed?` should be removed since it muddies the objective of this section with the previous one

Reply:
Copy/paste error, fixed.